config/config_test.go

```diff
@@ -212,7 +212,7 @@ allow-expression-index = true
 [isolation-read]
 engines = ["tiflash"]
 [security]
-spilled-file-encryption-method = "plaintext"
+data-encryption-method = "plaintext"
```

Comment:
what's the config value in TiKV and PD which indicates no encryption?

Reply:
It's the same as TiKV
https://github.com/tikv/tikv/blob/4547ac41e784ec7a9a119a5fd0d83140ae565d57/etc/config-template.toml#L804